test/clj/rems/application/test_commands.clj

```diff
@@ -97,16 +98,24 @@
                                         :visibility/field {:field/id "7"}
                                         :visibility/values ["y"]}}]}})
 
+(def dummy-resources
+  {1 {:id 1 :resid "res1"}
+   2 {:id 2 :resid "res2"}
+   3 {:id 3 :resid "res3"}
+   4 {:id 4 :resid "res4"}
+   5 {:id 5 :resid "res5"}
+   6 {:id 6 :resid "res-disabled" :enabled false}})
+
 (defn dummy-get-catalogue-item [id]
   (when (< id 10000)
     (some->> id
-             (getx {1 {:resid "res1"}
-                    2 {:resid "res2"}
-                    3 {:resid "res3" :formid 2}
-                    4 {:resid "res4" :wfid 2}
-                    5 {:resid "res5" :formid 2 :wfid 2}
-                    6 {:resid "res5" :formid nil}
-                    7 {:resid "res-disabled" :enabled false}
+             (getx {1 {:resource-id 1}
+                    2 {:resource-id 2}
+                    3 {:resource-id 3 :formid 2}
+                    4 {:resource-id 4 :wfid 2}
+                    5 {:resource-id 5 :formid 2 :wfid 2}
+                    6 {:resource-id 5 :formid nil}
+                    7 {:resource-id 6 :enabled false}
```

Comment:
Maybe sometime later we can get to `:resource/id`, `:form/id` and `:workflow/id` for all these.